Integrations/SplunkPy/SplunkPy.py

```diff
@@ -67,52 +67,20 @@ def get_current_splunk_time(splunk_service):
     raise ValueError('Error: Could not fetch Splunk time')
 
 
-def modify_str(matchobj):
-    any_str = matchobj.group(0)
-    new_str = any_str.replace(r'"', r'\"')
-    return new_str
-
-
 def rawToDict(raw):
     result = {}  # type: Dict[str, str]
-    updated_str = re.sub('\S+="(.*?)"', modify_str, raw, count=len(re.findall('\S+="(.*?)"', raw))-1)
-    final_str = re.sub('(\S+=)"(.*?")', r'\1\"\2', updated_str)
-    if final_str.startswith('{'):
-        new_raw = json.loads(final_str)
-        for element in new_raw.items():
-            val = element[1]
-            val = val.strip("\\")
-            val = val.strip("\"")
-            val = val.strip("\\")
-            key = element[0].strip()
-
-            alreadyThere = False
-            for dictkey, dictvalue in result.items():
-                if dictkey == key:
-                    alreadyThere = True
-                    result[dictkey] = dictvalue + "," + val
-
-            if not alreadyThere:
-                result[key] = val
-
-        parsed_raw = {}
-        raw_value = result.get('_raw', '')
-        for key_val in re.split('\S,', raw_value):
-            key_val = key_val.strip()
-            if '=' in key_val:
-                key_and_val = key_val.split('=')
-                parsed_raw[key_and_val[0]] = key_and_val[1]
-        result.update(parsed_raw)
-
-        return parsed_raw
+    raw_ = re.split('\S,', raw)
+    for key_val in raw_:
+        key_val = key_val.replace('"', '')
```

Comment:
do both `replace` and `strip` in the same line, and don't assign and initialize same arg, use different arg name